pyoverkiz/client.py

```diff
@@ -82,25 +64,26 @@ class OverkizClient:
     event_listener_id: str | None
     session: ClientSession
 
+    @staticmethod
+    def get_client(server: OverkizServer) -> OverkizClient:
+        return CLIENTS.get(server, SimpleClient)
+
     def __init__(
         self,
         username: str,
```

Comment:
Eventually we also need to think how we handle this, since local auth doesn't require a username / password.

Reply:
That's good to know.